### PR TITLE
[Serializer] documentation for allow_extra_attributes

### PR DESCRIPTION
**Pull request by juliendidier**
Doc PR for [symfony/symfony#19958](https://github.com/symfony/symfony/pull/19958)

### CODE REVIEW

File: components/serializer.rst [hunk 1]
```diff
@@ -151,6 +151,28 @@ needs three parameters:
 #. The name of the class this information will be decoded to
 #. The encoder used to convert that information into an array
 
+Extra attributes, attributes which are not mapped to the object, are allowed per default
+by the serializer.
+:method:`Symfony\\Component\\Serializer\\Serializer::deserialize`
+has a fourth parameter, the context. If you want to throw an exception if an attribute is outside
+the object scope, use ``allow_extra_attributes`` as ``false``.
+
+This will throws a :class:`Symfony\\Component\\Serializer\\Exception\\ExtraAttributesException` exception,
```
Comment (xabbuh) [inline components/serializer.rst]: This will throw an [...]

File: components/serializer.rst [hunk 2]
```diff
@@ -151,6 +151,28 @@ needs three parameters:
...
+because city is not an attribute of ``Acme\Person``::
```
Comment (xabbuh) [inline components/serializer.rst]: [...] because city is not an attribute of the `Person` class

File: components/serializer.rst [hunk 3]
```diff
@@ -151,6 +151,28 @@ needs three parameters:
 #. The name of the class this information will be decoded to
 #. The encoder used to convert that information into an array
 
+Extra attributes, attributes which are not mapped to the object, are allowed per default
+by the serializer.
+:method:`Symfony\\Component\\Serializer\\Serializer::deserialize`
+has a fourth parameter, the context. If you want to throw an exception if an attribute is outside
+the object scope, use ``allow_extra_attributes`` as ``false``.
```
Comment (xabbuh) [inline components/serializer.rst]: I would reword this a bit:

> By default, additional attributes that are not mapped to the denormalized object will be ignored by the Serializer component. Set the `allow_extra_attributes` key of the deserialization context to `false` to let the serializer throw an exception when additional attributes are passed.

File: components/serializer.rst (hunk 1 above)
Comment (juliendidier) [inline components/serializer.rst]: done

File: components/serializer.rst (hunk 2 above)
Comment (juliendidier) [inline components/serializer.rst]: done

File: components/serializer.rst (hunk 3 above)
Comment (juliendidier) [inline components/serializer.rst]: done

**Comment by juliendidier** [comment]
@xabbuh thanks for your review

**Review by dunglas — APPROVED** [review]

**Comment by xabbuh** [comment]
The code PR has been merged.

**Comment by wouterj** [comment]
👍 
status: reviewed

(there is a minor issue with the line length and the second paragraph can maybe replaced by a code comment in de code itself, but those things can be fixed during the merge)

**Comment by xabbuh** [comment]
Thank you @juliendidier.
